### PR TITLE
New Website

### PR DESCRIPTION
**Pull request by tailaiwang**
https://www.figma.com/file/VDb1JPEAKRiiZf3pnBw7TW/GoldenHack-2020?node-id=52%3A0
^Designs
Finished all pages except for # Last Year We Had... (Need help with ellipses, unsure of how to proceed)
Need to fix FAQ cards, need stock photos.

### CODE REVIEW

**Comment by tailaiwang** [comment]
Accidentally deleted yarn.lock, please disregard for now

**Comment by tailaiwang** [comment]
cd doc

File: src/index.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ const routing = (
       <Route path="/forgotpasswordpagesend" component={ForgotPasswordPageSend} />
       <Route path="/forgotpasswordpageinput" component={ForgotPasswordPageInput} />
       <Route path="/forgotpasswordpagechange" component={ForgotPasswordPageChange} />
+      <Route path="/websitepage" component={WebsitePage}/>
```
Comment (dunja-tomic) [inline src/index.js:29]: can we change this to `<Route path="/" component={WebsitePage}/>`?

Comment (tailaiwang) [reply src/index.js:29]: Yeah I was just unsure of what "HomePage" was, the current path="/" routes to that.

**Pull request merged**